build.rs

```diff
@@ -23,6 +23,8 @@ impl ParseCallbacks for RedisModuleCallback {
                 name: "isize",
                 is_signed: true,
             })
+        } else if name.starts_with("REDISMODULE_NOTIFY_") {
```

Comment:
Shouldn't this be handled the same as all the other enum discriminants and also be `isize`?

Reply:
Possibly -- I'm not familiar enough with the `bitflags` macro to be sure if it all fits together. The `IntKind::Int` matches the `c_int` used in the macro. These two must match up. If you use `isize` and then coerce it to `c_int` in the macro, that's fine too.

Reply:
ok, LGTM :+1: